protocol/test/forge/v2/strategies/RamosStrategy.t.sol

```diff
@@ -378,8 +384,28 @@ contract RamosStrategyTestVaultFunctions is RamosStrategyTestBase {
         strategy.borrowProtocolToken(amount, alice);
     }
 
-    function test_borrowProtocolToken() public {
-        // @todo
+    function test_borrowProtocolToken() public {       
+        vm.startPrank(executor);
+        // check balance before borrowing temple
+        assertEq(temple.balanceOf(address(trv)), 0);
+        assertEq(temple.balanceOf(address(ramos)), 0);
+        assertEq(temple.totalSupply(), TEMPLE_INITIAL_TOTAL_SUPPLY - TRV_STARTING_BALANCE);
```

Comment:
nit: The `TRV_STARTING_BALANCE` refers to DAI here, not temple I think?

Perhaps `TEMPLE_INITIAL_TOTAL_SUPPLY` should just be `37.xxxe25` above and then this should't do the subtraction?

Same in `test_repayProtocolToken`